lib/dao.js

```diff
@@ -2450,128 +2493,108 @@ DataAccessObject.prototype.updateAttributes = function updateAttributes(data, op
   Model.notifyObserversOf('before save', context, function(err, ctx) {
     if (err) return cb(err);
     data = ctx.data;
-
```

Comment:
This empty line should be preserved as a visual delimiter. I'll fix this myself to speed things up.